apps/src/storage/dataBrowser/dataVisualizer/ChartPlaceholder.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import color from '../../../util/color';
+
+const styles = {
+  container: {position: 'relative', textAlign: 'center'},
+  text: {
+    position: 'absolute',
+    width: '100%',
+    bottom: '50%',
+    fontFamily: '"Gotham 5r", sans-serif, sans-serif',
+    fontSize: 20,
+    color: color.dark_charcoal
+  }
+};
+
+class ChartPlaceholder extends React.Component {
+  render() {
+    return (
+      <div style={styles.container}>
+        <div style={styles.text}>Select values to generate a visualization</div>
```

Comment:
I18n

Reply:
Nothing for datasets has been i18n-ized yet. Planning on doing that once things are a little more finalized, since a lot of these strings have changed a couple times. Definitely will be done before this comes out of experiment.